### PR TITLE
For the --minconfig subcommand of edgecore and edgesite, hide the database config:  DriverName and AliasName

### PR DESCRIPTION
**Pull request by kuramal**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind cleanup

**What this PR does / why we need it**:
For the --minconfig subcommand of edgecore and edgesite, hide the database config:  DriverName and AliasName, which are not required for min config

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/edgesite/v1alpha1/default.go [hunk 1]
```diff
@@ -141,8 +141,6 @@ func NewMinEdgeSiteConfig() *EdgeSiteConfig {
 			APIVersion: path.Join(GroupName, APIVersion),
 		},
 		DataBase: &edgecoreconfig.DataBase{
-			DriverName: DataBaseDriverName,
```
Comment (kadisi) [inline pkg/apis/edgesite/v1alpha1/default.go]: We need to do the same for edgecore

File: pkg/apis/edgesite/v1alpha1/default.go [hunk 2]
```diff
@@ -157,6 +155,7 @@ func NewMinEdgeSiteConfig() *EdgeSiteConfig {
 				ClusterDNS:            "",
 				ClusterDomain:         "",
 				RemoteRuntimeEndpoint: constants.DefaultRemoteRuntimeEndpoint,
+				//TODO (@kuramal) auto set the value of PodSandboxImage by the architecture(x86,amd64,arm or arm64)
```
Comment (fisherxu) [inline pkg/apis/edgesite/v1alpha1/default.go]:
```suggestion
				//TODO (@kuramal) Automatically set PodSandboxImage according to the architecture.(x86,amd64,arm or arm64)
```
And please move this line above L160

File: pkg/apis/edgesite/v1alpha1/default.go [hunk 3]
```diff
@@ -157,6 +155,7 @@ func NewMinEdgeSiteConfig() *EdgeSiteConfig {
 				ClusterDNS:            "",
 				ClusterDomain:         "",
 				RemoteRuntimeEndpoint: constants.DefaultRemoteRuntimeEndpoint,
+				//TODO (@kuramal) Automatically set PodSandboxImage according to the architecture.(x86,amd64,arm or arm64)
```
Comment (fisherxu) [inline pkg/apis/edgesite/v1alpha1/default.go]: move this line above L160

**Pull request merged**